experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -44,23 +46,28 @@ class Flowables {
   }
 
   template <typename T>
-  static Reference<Flowable<T>> just(std::initializer_list<T> list) {
-    auto lambda = [ list, it = list.begin() ](
+  static Reference<Flowable<T>> justN(std::initializer_list<T> list) {
+    std::vector<T> vec;
+    std::move(list.begin(), list.end(), std::back_inserter(vec));
+
+    size_t i = 0;
```

Comment:
Does it have to be in the capture list? Why not just define it inside the lambda func - like emitted & done?